### PR TITLE
Beam tags publishing

### PR DESCRIPTION
**Pull request by quininez**
## Description
* add tags on beam publishing
* display tags on beams

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by josenriagu — DISMISSED** [review]

File: ui/hooks/src/utils/entry-utils.ts [hunk 1]
```diff
@@ -124,5 +124,6 @@ export const mapBeamEntryData = (
     createdAt: beam.createdAt,
     content: beam.content,
     nsfw: beam?.nsfw,
+    tags: beam?.tags?.map(labeledTag => labeledTag.value),
```
Comment (kenshyx) [inline ui/hooks/src/utils/entry-utils.ts]: I think this requires first a filter over the `labelType` to be of `sdk.current.services.gql.labelTypes.TAG`

Comment (quininez) [reply ui/hooks/src/utils/entry-utils.ts]: Only tags should be there, but I can add the filter as an extra safeguard

File: ui/hooks/src/utils/entry-utils.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Comment } from '@akashaorg/typings/lib/sdk/graphql-types';
 import type { PostResultFragment } from '@akashaorg/typings/lib/sdk/graphql-operation-types';
 import type { BeamEntryData, ReflectEntryData } from '@akashaorg/typings/lib/ui';
 import { AkashaBeam, AkashaReflect } from '@akashaorg/typings/lib/sdk/graphql-types-new';
+import getSDK from '@akashaorg/awf-sdk';
 
 export const MEDIA_URL_PREFIX = 'CID:';
 export const TEXTILE_GATEWAY = 'https://hub.textile.io/ipfs/';
```
Comment (kenshyx) [inline ui/hooks/src/utils/entry-utils.ts]: unrelated to the pr changes, can we remove this?

Comment (quininez) [reply ui/hooks/src/utils/entry-utils.ts]: yes, they are no longer used

**Review by kenshyx — APPROVED** [review]

**Pull request merged**